server/src/resolvers/Join.ts

```diff
@@ -34,13 +34,29 @@ export const updateJoin: FieldResolver<'Mutation', 'updateJoin'> = async (
           {
             where: { id: join!.tables[0].id },
             data: {
+              owner:
+                data.tables && data.tables[0].owner
+                  ? {
+                      connect: {
+                        id: data.tables[0].owner.id,
+                      },
+                    }
+                  : undefined,
```

Comment:
Is the following clearer?
```
...(data.tables && data.tables[0].owner && {
    owner: {
        connect: { id: data.tables[0].owner.id }
    },
})
```

Reply:
I don't think so, it's shorter but I find it more complex because destructuring a null object in order to avoid having the "owner" field in the object is trickier than a ternary (imo)